src/prospectScraper/Program.cs

```diff
@@ -117,12 +145,37 @@ private static void RunTheMockDraft()
             //TODO - Implement Mock Draft
             var webGet = new HtmlWeb();
             webGet.UserAgent = "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:31.0) Gecko/20100101 Firefox/31.0";
-            var document1 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1.asp");
-            var document2 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1b.asp");
-            var document3 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-2.asp");
-            var document4 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-3.asp");
-            var document5 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-4.asp");
-            var document6 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-6.asp");
+            
+            // Generate all the needed URL's
+            string[] URLs = GenerateDataURLs(
+                "https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-<PageNo>.asp",
+                "<PageNo>", 1, 6, new string[]{"1b"}, new string[]{});
```

Comment:
I don't think this works out as it did previously.

The order should be 

- 1
- 1b
- 2

etc.

It seems like the way it works currently is going from 1-6 then 1b, putting picks 17-32 at the end instead of between 16 and 33.